scripts/verify-structure.py

```
@@ -68,104 +206,139 @@
+            f"{context}: INTERNAL ERROR - Invalid schema definition: {e}\n"
+            f"  This is a bug in the verification script, please report it"
+        )
+        return errors
+
+    try:
+        for error in validator.iter_errors(data):
+            path = " -> ".join(str(p) for p in error.path) if error.path else "root"
+            errors.append(f"{context}: {path}: {error.message}")
+    except RecursionError:
+        errors.append(f"{context}: Data structure too deeply nested (recursion limit)")
+    except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)